app/src/main/java/com/android/unio/ui/event/EventCard.kt

```diff
@@ -167,6 +165,10 @@ fun EventCard(
       shouldBeEditable = shouldBeEditable)
 }
 
+private operator fun Any.getValue(nothing: Nothing?, property: KProperty<*>): Any {
+  TODO("Not yet implemented")
+}
```

Comment:
Unused function If I'm not mistaken? and no TODOs in the code please

Reply:
Oh I did not see this. I think Android Studio automatically added this. Thank you !